OHarmash_HW/Teachers-And-Students/Teachers-And-Students/PrintInfo.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Teachers_And_Students
+{
+    internal static class PrintInfo
+    {
+        
+        internal static void ShowGeneralStat()
+        {
+            Console.Clear();
+
+            var assistantStudent = Groups.Assistant.Count;
+            var lecturerStudent = Groups.Lecturer.Count;
+            var docentStudent = Groups.Docent.Count;
+            var tab = "";
+
+            Console.WriteLine("Assistant {0, -1} has {1, -3} students in group", tab, assistantStudent);
+            Console.WriteLine("Lecturer {0, -2} has {1, -3} students in group", tab, lecturerStudent);
+            Console.WriteLine("Docent {0, -4} has {1, -3} students in group \n", tab, docentStudent);
```

Comment:
code duplication 
move to the separate method with corresponded params

Reply:
Got your idea. But in that case, we have different word length, it's just for text formating